### ISSUE TITLE
Can't download dataset with university email

### ISSUE THREAD

**Issue by selmadeac** [scaleapi/pandaset-devkit#90]
Hi! 

I am a PhD student working as a researcher at Technical University of Cluj-Napoca. Although I offer my uni email, an email with a link to download the dataset is not sent to me. Why is this so? Only company emails work? Or has the dataset become unavailable?

**Comment by AlfredQin** [comment]
Hello, @selmadeac , I am also a student and I meet the same problem with you. Do you resolve the problem? Thank you

**Comment by utkarshani** [comment]
Same here. I am unable to download the dataset too. I did not receive the email with the download link.

**Comment by TrinhTuanHung2021** [comment]
I used gmail and downloaded it very easily

**Comment by TrinhTuanHung2021** [comment]
> Hi!
> 
> I am a PhD student working as a researcher at Technical University of Cluj-Napoca. Although I offer my uni email, an email with a link to download the dataset is not sent to me.
> 
> I can successfully download datasets from KITTI, Waymo, Lyft and even the Audi dataset using this email, but not yours. Why is this so? Only company emails work? Or has the dataset become unavailable?

Have you downloaded it?

**Comment by selmadeac** [comment]
I did not try lately because now I am working on another project where I no longer need it. I used another dataset in that project, in the end. @utkarshani ,  @AlfredQin did you manage to download it in the end?

**Comment by utkarshani** [comment]
@selmadeac Yes, I reached out to Support@scale.com, or nofel.naoman@scale.com, or shaun@scale.com.
If you are still facing trouble reach out the above email. They are very helpful.

**Comment by FloatingPoint64** [comment]
Hi there

Is this issue solved?  
I try to download the dataset today and success to get it.  
I accessed it from this URL and used G-mail.  
https://pandaset.org
